cmd/run_controller/main.go

```diff
@@ -27,6 +27,7 @@ func init() {
 
 	flag.IntVar(&burst, "burst", 10, "burst for RESTClient")
 	flag.IntVar(&qps, "qps", 5, "QPS for RESTClient")
+	flag.IntVar(&threadiness, "threadiness", 2, "number of workers to be started each in a single thread")
```

Comment:
My proposal:

```suggestion
	flag.IntVar(&threadiness, "threadiness", 2, "maximum number of reconciliations performed in parallel")
```

Reply:
Thanks for your proposal. It sounds more meaningful now